images/cellranger/install-bcl2fastq.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
I think we could delete this file since we no longer use it correct?

Reply:
I decided to leave it, mostly for reference in case we decide we can't use the rpm in the future.

Reply:
Makes sense 👍